test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -44,6 +44,11 @@ def can_run_smdataparallel(ecr_image):
     ) >= Version("110")
 
 
+def skip_unsupported_instances_smmodelparallel(instance_type):
```

Comment:
nit: Should we change it to `skip_unsupported_instances_smdataparallel` for this file? (Not a merge blocker)

Reply:
Renamed function.